### PR TITLE
Added 6 rooks codesandbox examples

### PR DESCRIPTION
**Pull request by mahijendra**
Appended six rooks codesandbox examples

### CODE REVIEW

File: docusaurus/docs/useBoundingclientrect.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+s---
```
Comment (imbhargav5) [inline docusaurus/docs/useBoundingclientrect.md:1]: This is a typo

File: docusaurus/docs/useWillUnmount.md [hunk 2]
```diff
@@ -1,11 +1,7 @@
 ---
-id: useWillUnmount
-title: useWillUnmount
-sidebar_label: useWillUnmount
+id: useWillUnmount title: useWillUnmount sidebar_label: useWillUnmount
```
Comment (imbhargav5) [inline docusaurus/docs/useWillUnmount.md:2]: These 2 must be in 2 lines

**Comment by imbhargav5** [comment]
Also it might be a good idea to rebase from the main branch, since there are merge conflicts.

**Comment by imbhargav5** [comment]
@mahijendra Can you confirm if all the examples you made have already been added to the https://react-hooks.org website?
If some haven't been added let's create a new PR with the ones missing.

**Comment by mahijendra** [comment]
> @mahijendra Can you confirm if all the examples you made have already been added to the https://react-hooks.org website?
> If some haven't been added let's create a new PR with the ones missing.

Yeah sure! will do

> @mahijendra Can you confirm if all the examples you made have already been added to the https://react-hooks.org website?
> If some haven't been added let's create a new PR with the ones missing.

Alright sure! I believe there are 3 rooks examples I made that haven't been added to the website.

**Comment by imbhargav5** [comment]
@mahijendra alright let's create a new PR with just those and get it merged.

**Comment by mahijendra** [comment]
> @mahijendra alright let's create a new PR with just those and get it merged.

I've created a PR already

**Comment by qiweiii** [comment]
All examples have been added

**Pull request closed without merging**